library/Module/Discover/Run.hs

```diff
@@ -22,40 +22,45 @@ import System.FilePattern.Directory (FilePattern, getDirectoryFiles)
 import System.IO (hPutStrLn, stderr)
 
 run
-  :: ([Module] -> IO String) -- ^ Output function
-  -> Maybe FilePattern -- ^ Optional file Pattern
-  -> [String] -- ^ Processor arguments
+  :: ([Module] -> IO String)
+  -- ^ Output function
+  -> Maybe FilePattern
+  -- ^ Optional file Pattern
+  -> [String]
+  -- ^ Processor arguments
   -> IO ()
 run output filePattern args = do
   case args of
     src : _ : dst : _args -> do
       let srcDir = takeDirectory src
-      modules <- fmap (toModule srcDir)
-        <$> getDirectoryFiles srcDir [fromMaybe "**/*.hs" filePattern]
+      modules <-
+        fmap (toModule srcDir)
+          <$> getDirectoryFiles srcDir [fromMaybe "**/*.hs" filePattern]
       out <- output modules
       writeFile dst out
     _ -> do
       hPutStrLn stderr "unexpected arguments"
       exitFailure
 
 toModule :: FilePath -> FilePath -> Module
-toModule srcDir filePath = Module
-  { moduleName = dropExtension $ takeFileName filePath
-  , moduleFullName = toModuleFullName srcDir filePath
-  , moduleFilePath = srcDir </> filePath
-  }
+toModule srcDir filePath =
+  Module
+    { moduleName = dropExtension $ takeFileName filePath
+    , moduleFullName = toModuleFullName srcDir filePath
+    , moduleFilePath = srcDir </> filePath
+    }
 
 toModuleFullName :: FilePath -> FilePath -> String
 toModuleFullName srcDir filePath =
-  replace [pathSeparator] "."
-    $ dropExtension
-    $ joinPath
-    $ reverse
-    $ takeWhile (isUpper . head)
-    $ reverse
-    $ splitPath
-    $ srcDir
-    </> filePath
+  replace [pathSeparator] "." $
+    dropExtension $
+      joinPath $
+        reverse $
+          takeWhile (isUpper . head) $
+            reverse $
+              splitPath $
+                srcDir
+                  </> filePath
```

Comment:
I've _seen_ Fourmolu do operator first here, I know I have. But then not always. I think it keeps it operator first in megarepo? Frustrating.

Reply:
This seems like something I just want to rewrite as a function composition chain anyway.